yarn.lock

```diff
@@ -1865,17 +1859,17 @@ tslint@^3.15.1:
     resolve "^1.1.7"
     underscore.string "^3.3.4"
 
-type-detect@^1.0.0:
-  version "1.0.0"
-  resolved "https://registry.yarnpkg.com/type-detect/-/type-detect-1.0.0.tgz#762217cc06db258ec48908a1298e8b95121e8ea2"
-
 type-detect@0.1.1:
   version "0.1.1"
   resolved "https://registry.yarnpkg.com/type-detect/-/type-detect-0.1.1.tgz#0ba5ec2a885640e470ea4e8505971900dac58822"
 
-typescript@^2.0.0:
-  version "2.0.3"
```

Comment:
I figured this out, I think.  If you just run "yarn" on ToT with no other changes, it recreates the .lock file (I guess because we modified package.json?).  And that updated .lock file gets the new version.